tests/smoke/rest.smoke.test.ts

```diff
@@ -708,7 +712,7 @@ describe('HiFi API REST Calls', () => {
             expect(responseJSON).toEqual([attenuation3Data]);
```

Comment:
Hmm, if this is working, then I think the new documentation is wrong. It currently says the ID key is "zone-attenuation-id". I suspect this is a regression that I introduced.